### PR TITLE
New Post: Announcing Subvisual Live Talks

### PR DESCRIPTION
**Pull request by frm**

### CODE REVIEW

File: src/posts/2020-04-17-announcing-subvisual-live-talks/index.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+author: fernando-mendes
+date: 2020-04-17
+intro: >
+  A while ago I felt the shame of copying and pasting code around from projects.
+  Eventually, I decided to stop being lazy, extracting it and making a hex
+  package out of it. This is story of that project and a walkthrough all the
+  awesome things it does, with very little code. Yay Elixir!
+---
+
+# Announcing Subvisual Live Talks: Zamith, Josh Clayton, Saša Jurić and others
```
Comment (pfac) [inline src/posts/2020-04-17-announcing-subvisual-live-talks/index.md:14]: The title is rendered from the value in the frontmatter, this is hidden.

File: src/posts/2020-04-17-announcing-subvisual-live-talks/index.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+id: 148
+path: /posts/148-announcing-subvisual-live-talks/
+title: "Announcing Subvisual Live Talks: Zamith, Josh Clayton, Sasa Juric and others"
+author: fernando-mendes
+date: 2020-04-17
+intro: >
+  A while ago I felt the shame of copying and pasting code around from projects.
+  Eventually, I decided to stop being lazy, extracting it and making a hex
+  package out of it. This is story of that project and a walkthrough all the
+  awesome things it does, with very little code. Yay Elixir!
```
Comment (pfac) [inline src/posts/2020-04-17-announcing-subvisual-live-talks/index.md:11]: This is the intro from your previous post about Dictator.

Comment (frm) [reply src/posts/2020-04-17-announcing-subvisual-live-talks/index.md:11]: I f'ed up the copy/paste. Shame. Thanks!

File: src/posts/2020-04-17-announcing-subvisual-live-talks/index.md (hunk 1 above)
Comment (frm) [reply src/posts/2020-04-17-announcing-subvisual-live-talks/index.md:14]: Yeah, copy pasted from Typora again. Cheers!

**Pull request merged**